ang/civicase/CaseDetails.js

```diff
@@ -266,7 +266,16 @@
       return ret;
     }
 
+    /**
+     * Watches for case changes. When the case is locked it redirects the user
+     * to the case list. Also, If the case is loaded without its definition, it
+     * will make a request to get the missing information.
+     */
     function itemWatcher () {
+      if ($scope.item && $scope.item.lock) {
```

Comment:
Please move this to a self containing function called `redirectToCaseListWhenTryingToAccessLockedCase` or something like that.